recipe/recipe.yaml

```diff
@@ -191,7 +191,7 @@ outputs:
   - package:
       name: torchvision-tests
     build:
-      script: true
+      script: ${{ 'true' if unix else 'exit /b 0' }}
```

Comment:
i'm having a hard time reading this mixed jinja / v1 recipe syntax.

does this mean that the tests are entirely skipped on windows? this isn't seem right.

Reply:
This is not a test, this is the build script of the package. I think this is to override the automatic use of `build.sh/build.bat`.

Reply:
This has nothing to do with tests, but with the build scripts. I don't know this aspect of the v1 recipe in detail though - perhaps @Tobias-Fischer or @wolfv can comment?

Reply:
(did your comment get crossed with his)

Reply:
Yes, our comments crossed...

Reply:
> I don't know this aspect of the v1 recipe in detail though - perhaps @Tobias-Fischer or @wolfv can comment?

What I meant: the whole `script.build` aspect seems kind of pointless here - why do we have it in the recipe at all?

Reply:
Because otherwise it picks up build.sh or build.bat

Reply:
That is a massive antipattern. Please let outputs opt into build scripts explicitly (yes, even if they need to specify `build.sh` / `build.bat`), and don't ask script-less outputs to opt out of a magic and useless default.

PS. I've made this request a couple months already, can't search where on a phone right now.

Reply:
Want to make a PR? :) 

It's the same behavior as conda build but I'm open to change it.

Note that you can also test PKG_NAME in a build script

Reply:
Will put it on my to-do list. ;)